src/components/form/form.jsx

```diff
@@ -128,3 +132,74 @@ Form.Feedback = ({ className, message }) => {
     </>
   );
 };
+
+Form.Combobox = ({ id, labelText, className, list, name, nameKey, idKey }) => {
+  const [selectedItems, setSelectedItems] = useState([]);
+  const [query, setQuery] = useState("");
+  const filteredList =
+    query === ""
+      ? list
+      : list.filter((item) => {
+          return item[nameKey].toLowerCase().includes(query.toLowerCase());
+        });
+  return (
+    <div className={`flex font-bold font-arial flex-col py-3 ${className}`}>
+      <label htmlFor={id} className={`mb-1.5 text-lg text-start `}>
+        {labelText}
+      </label>
+      <Combobox
+        value={selectedItems}
+        onChange={setSelectedItems}
+        multiple
+        nullable
+        name={name}
+      >
+        <div className=" relative font-normal py-3 px-4 border-solid rounded-none border border-black focus-within:ring-black focus-within:ring-1 flex gap-2.5 flex-wrap">
+          {selectedItems.length > 0 &&
+            selectedItems.map((item) => (
+              <div
+                key={item[idKey]}
+                className="flex gap-1.5 bg-neutral-300 px-1.5 py-1"
+              >
+                <span>{item[nameKey]}</span>
+                <button
+                  type="click"
+                  onClick={() => {
+                    setSelectedItems(
+                      selectedItems.filter(
+                        (element) =>
+                          element[idKey] !== item[idKey] ||
+                          element[nameKey] !== item[nameKey]
+                      )
+                    );
+                  }}
+                >
+                  <img src={cross} />
+                </button>
+              </div>
+            ))}
+          <Combobox.Input
+            onChange={(event) => setQuery(event.target.value)}
+            value={query}
+            className="font-nomral border-none focus:outline-none flex-grow flex-shrink-0 w-16"
+          />
+          <Combobox.Button className="absolute right-5 flex top-4">
+            <img src={down} className="h-2.5 w-2.5" />
+          </Combobox.Button>
+        </div>
+        <Combobox.Options>
+          {filteredList.map((item) => (
+            <Combobox.Option
+              key={item[idKey]}
+              value={item}
+              className="font-normal py-3 px-4 border-solid rounded-none border border-black"
```

Comment:
Adding the tailwind classes cursor-pointer and hover:bg-(some color) will help dramatical with the user experience here

Reply:
Even though the options are displayed in the input of the combobox, adding a highlight to options that have been selected in the dropdown would be helpful as well. This way the user can see which options they've already selected.